tests/conftest.py

```diff
@@ -140,12 +140,14 @@ def flask_test_client(app, user_token=None):
     :return: A flask test client.
     """
     with app.test_client() as test_client:
-        test_client.set_cookie(
-            "localhost",
-            "fsd_user_token",
-            user_token or create_valid_token(),
-        )
-        yield test_client
+        try:
+            test_client.set_cookie(
+                "fsd_user_token",
+                user_token or create_valid_token(),
+            )
+            yield test_client
+        finally:
+            test_client.set_cookie("fsd_user_token", "")
```

Comment:
Not an issue, but curious to know why we need to unset the cookie? I assume `test_client` can't be used after it returns from the yield anyway .. ?

Reply:
Removed the unnecessary finally block.